CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 All notable changes to this project will be documented in this file. This project adheres to Semantic Versioning.
 
+### 5.4.1
+* Fix product availability building for products with OOS threshold
```

Comment:
Better change it to: 
Fix product availability issue where product shows as invisible